perun/collect/complexity/strategy.py

```diff
@@ -0,0 +1,146 @@
+""" Module with complexity collector strategies for tracing.
+
+    The strategies are meant to be useful default settings for collection so that user
+    does not have to specify every detail for each collection. The strategies focus on
+    userspace / everything collection with sampling / no sampling etc.
+
+    The assemble_script method serves as a interface which handles specifics of each strategy.
+"""
+
+import collections
+
+import perun.utils.exceptions as exceptions
+
+# The default global sampling for 'sample' strategies if global sampling is not set
+_DEFAULT_SAMPLE = 5
+
+
+def get_supported_strategies():
+    """Provides list of supported strategies.
+
+    :returns: list -- the names of supported strategies
+    """
+    return [key for key in _STRATEGIES.keys()]
```

Comment:
Doesn't "list(_STRATEGIES.keys())" provide the same functionality and is clearer?